deploy/debezium-connector-hosts.yml

```diff
@@ -0,0 +1,74 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: kafka-connectors
+objects:
+- apiVersion: kafka.strimzi.io/v1beta2
+  kind: KafkaConnector
+  metadata:
+    name: hbi-migration-connector
+    annotations:
+      strimzi.io/use-connector-resources: "true"
+    labels:
+      strimzi.io/cluster: ${KAFKA_CONNECT_INSTANCE}
+  spec:
```

Comment:
minor change: i was thinking it would be good to deploy these by default in a stopped state so we can control when things run

setting is 
```yaml
spec:
  state: stopped
```